cms/djangoapps/contentstore/tests/test_crud.py

```diff
@@ -69,42 +48,49 @@ def test_get_some_templates(self):
         self.assertIsNotNone(HtmlDescriptor.get_template('announcement.yaml'))
 
     def test_factories(self):
-        test_course = persistent_factories.PersistentCourseFactory.create(
-            course='course', run='2014', org='testx',
-            display_name='fun test course', user_id='testbot'
+        test_course = CourseFactory.create(
+            org='testx',
+            course='course',
+            run='2014',
+            display_name='fun test course',
+            user_id='testbot'
         )
         self.assertIsInstance(test_course, CourseDescriptor)
         self.assertEqual(test_course.display_name, 'fun test course')
-        index_info = self.split_store.get_course_index_info(test_course.id)
-        self.assertEqual(index_info['org'], 'testx')
-        self.assertEqual(index_info['course'], 'course')
-        self.assertEqual(index_info['run'], '2014')
+        course_from_store = self.store.get_course(test_course.id)
+        self.assertEqual(course_from_store.id.org, 'testx')
+        self.assertEqual(course_from_store.id.course, 'course')
+        self.assertEqual(course_from_store.id.run, '2014')
 
-        test_chapter = persistent_factories.ItemFactory.create(
-            display_name='chapter 1',
-            parent_location=test_course.location
+        test_chapter = ItemFactory.create(
+            parent_location=test_course.location,
+            category='chapter',
+            display_name='chapter 1'
         )
         self.assertIsInstance(test_chapter, SequenceDescriptor)
         # refetch parent which should now point to child
-        test_course = self.split_store.get_course(test_course.id.version_agnostic())
+        test_course = self.store.get_course(test_course.id.version_agnostic())
         self.assertIn(test_chapter.location, test_course.children)
 
         with self.assertRaises(DuplicateCourseError):
-            persistent_factories.PersistentCourseFactory.create(
-                course='course', run='2014', org='testx',
-                display_name='fun test course', user_id='testbot'
+            CourseFactory.create(
+                org='testx',
+                course='course',
+                run='2014',
+                display_name='fun test course',
+                user_id='testbot'
```

Comment:
This test is wonky. What stuff is it actually trying to test? Why is it asserting multiple things in the same test function?

(Not your fault, I know. Your test is an authentic translation of the original).